packages/backend/src/openapi/webhooks.yaml

```diff
@@ -0,0 +1,275 @@
+openapi: 3.1.0
```

Comment:
Can we add an openapi validattion check to github actions a la
https://github.com/interledger/open-payments/blob/main/.github/workflows/validate-openapi.yaml

Reply:
Yes, I'll add that.